content/dart/concepts/map/map.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Map'
+Description: 'The Map object is a simple key/value pair. Keys and values in a map may be of any type. A Map is a dynamic collection. In other words, Maps can grow and shrink at runtime.'
```

Comment:
Give a small and breif desc here. This big definition about map can go below this table(meta data).